requirements.in

```diff
@@ -9,6 +9,7 @@ slugify
 python-dateutil
 invoke
 colored
+click
```

Comment:
Looks like this is unused, can we revert all the requirements changes?

Reply:
have removed this now :)